tools/devtool

```diff
@@ -861,6 +861,8 @@ cmd_test_debug() {
 #
 cmd_fmt() {
     cmd_sh "cargo fmt --all -- --config $(tr '\n' ',' <tests/fmt.toml)"
+    cmd_sh "black tests tools .buildkite"
```

Comment:
Does black pick up its configuration when run like this? I usually do `cd tests; black . ../.buildkite ../tools`

Reply:
Ah, good point. Updated!